cockroachcloud/migrations-page.md

```diff
@@ -224,57 +252,55 @@ To edit a statement, click the **Edit** button or the statement itself and enter
-| Suggestion                                       | Review and take any relevant actions indicated by the message. Then optionally check **Acknowledge**.                                                                                                                                                                                                                                                                                                                     | ✗                         |
+To export the current schema, click **Export SQL File** at the top of the [**Statements** list](#statements-list).
 
+## Update the schema
 
-After updating the schema, **you must [retry the migration](#retry-the-migration)** to update the **Summary Report**. This is necessary in order to verify that the schema has zero errors and can be [finalized](#finalize-the-schema).
+To update the schema, apply bulk actions in the [**Summary Report**](#summary-report) or edit, add, or remove statements in the [**Statements** list](#statements-list).
 
-To export the current schema, click **Export SQL File** at the top of the **Statements** list.
+|                       Type                       |                                                                                                                                                                                                          Solution                                                                                                                                                                                                         |   Bulk Actions   | Required for schema migration? |
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|--------------------------------|
+| Unimplemented feature                            | The feature does not yet exist on CockroachDB. Implement a workaround by editing the statement and adding statements. Otherwise, remove the statement from the schema. If a link to a tracking issue is included, click the link for further context. For more information about unimplemented features, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html#unimplemented-features). | Delete           | ✓                              |
```

Comment:
Style question that doesn't need to be decided upon in this PR: Are these checks/Xs accessible? They don't seem visually different enough (unless you make them green/red as well maybe) that they are preferable over just putting Yes/No. Do we have guidance for using these?